src/swell/utilities/slurm.py

```diff
@@ -15,18 +17,25 @@
 
 def prepare_scheduling_dict(
     logger: Logger,
-    experiment_dict: dict
+    experiment_dict: dict,
+    platform: str,
 ):
-    # Hard-coded defaults
-    # ----------------------------------------------
-    global_defaults = {
-        "account": "g0613",
-        "qos": "allnccs",
-        "nodes": 1,
-        "ntasks-per-node": 24,
-        "constraint": "cas|sky"
-    }
 
+    # Obtain platform-specific SLURM directives and set them as global defaults
+    # Start by constructing the full platforms path
+    # -------------------------------------------
+    platform_path = f"swell.deployment.platforms.{platform}"
+
+    # Import the path dynamically
+    # ------------------------------
+    path_import = __import__(platform_path, fromlist=[''])
```

Comment:
[Best practice](https://docs.python.org/3/reference/import.html#importlib) is to use `importlib.import_module` instead of `__import__` for this.

Also, good opportunity for better error handling.

```suggestion
    try:
        path_import = importlib.import_module(platform_path)
    except ModuleNotFoundError:
        raise Exception(f"Platform '{platform}' has not been configured in SWELL")
    except Exception as err:
        raise err
```